examples/ingest/airtable/ingest.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Processes all the documents in all bases (and all workspaces) within an Airtable org,
+# using the `unstructured` library.
+
+# Structured outputs are stored in airtable-ingest-output
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+cd "$SCRIPT_DIR"/../../.. || exit 1
+
+# Obtain your personal access token, save/source them from another file, for security reasons:
+# source "./../../secrets/airtable.txt"
```

Comment:
probably okay to omit this and let the user decide how this gets set. but is there a good doc to point to on HOW TO get the personal access token inside airtable?

Reply:
Removed that comment and added relevant documentation on how to get the token 👍

https://github.com/Unstructured-IO/unstructured/blob/64de00c20d213d99b3734c613d059245191cddc6/examples/ingest/airtable/ingest.sh#L10-L14